src/Components/Patient/PatientHome.tsx

```diff
@@ -1088,7 +1089,184 @@ export const PatientHome = (props: any) => {
+                    </span>
+                  </div>
+
+                  <div>
+                    <button
+                      className="btn btn-primary w-full text-xs"
+                      disabled={
+                        patientData.is_active && !!consultationListData.length
+                      }
+                      onClick={() =>
+                        navigate(
+                          `/facility/${patientData?.facility}/patient/${id}/consultation`
+                        )
+                      }
+                    >
+                      Add Consultation
+                    </button>
+                  </div>
+                </div>
+              </div>
+              <div className="w-full">
+                <div className="bg-white rounded-lg shadow p-4 h-full space-y-2">
+                  <div className="text-green-700 text-center">
+                    <span>
+                      <i className="fa-regular fa-file-lines fa-4x"></i>
+                    </span>
+                  </div>
+                  <div>
+                    <button
+                      className="btn btn-primary w-full text-xs"
+                      onClick={() =>
+                        navigate(`/patient/${id}/investigation_reports`)
+                      }
+                    >
+                      Investigations Summary
+                    </button>
+                  </div>
+                </div>
+              </div>
+              <div className="w-full">
+                <div className="bg-white rounded-lg shadow p-4 h-full space-y-2">
+                  <div className="text-green-700 text-center">
+                    <span>
+                      <i className="fa-solid fa-file-arrow-up fa-4x"></i>
+                    </span>
+                  </div>
+                  <div>
+                    <button
+                      className="btn btn-primary w-full text-xs"
+                      onClick={() =>
+                        navigate(
+                          `/facility/${patientData?.facility}/patient/${id}/files/`
+                        )
+                      }
+                    >
+                      View/Upload Patient Files
+                    </button>
+                  </div>
+                </div>
+              </div>
+              <div className="w-full">
+                <div className="bg-white rounded-lg shadow p-4 h-full space-y-2">
+                  <div className="text-green-700 text-center">
+                    <span>
+                      <i className="fas fa-ambulance fa-4x"></i>
+                    </span>
+                  </div>
+
+                  <div>
+                    <RoleButton
+                      className="btn btn-primary w-full text-xs"
+                      disabled={
+                        !patientData.is_active ||
+                        !(
+                          patientData?.last_consultation?.facility == facilityId
```

Comment:
```suggestion
                          patientData?.last_consultation?.facility === facilityId
```
https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=9682972